php_assin4.php

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Assignment 4</title>
+  <style>
+    .pagination {
+      display: inline-block;
+    }
+    .pagination a {
+      color: black;
+      float: left;
+      padding: 8px 16px;
+      text-decoration: none;
+    }
+    .pagination a.active {
+      background-color: #4CAF50;
+      color: white;
+    }
+    .pagination a:hover{
+      background-color: #ddd;
+    }
+  </style>
```

Comment:
Why is the styling of pagination in this file.....and try to use external CSS in future if you need to